lib/agoneum/account/account.ex

```diff
@@ -63,7 +63,7 @@ defmodule Agoneum.Account do
   """
   def create_user(attrs \\ %{}) do
     %User{}
-    |> User.changeset(attrs)
+    |> User.registration_changeset(attrs)
     |> Repo.insert()
   end
```

Comment:
![image](https://user-images.githubusercontent.com/1091716/28165370-82f7dde2-67a1-11e7-8347-91036a786e3e.png)
Looks like this structure is correct